### ISSUE TITLE
Add "idescriptive" linter

### ISSUE THREAD

**Issue by leonklingele** [golangci/golangci-lint#3723]
### Your feature request related to a problem? Please describe.

The `idescriptive` linter reports obscure interfaces, complaining about funcs whose in/out parameters are unnamed.

```go
type BadSingleParam interface {
	NoReturn(int)                          // want `missing name for incoming parameter int in method NoReturn\(int\)`
	SingleReturn(bool) (x int)             // want `missing name for incoming parameter bool in method SingleReturn\(bool\) \(x int\)`
	DoubleReturn(Empty) (y int, err error) // want `missing name for incoming parameter Empty in method DoubleReturn\(Empty\) \(y int, err error\)`
}
```

https://github.com/maratori/idescriptive

### Describe the solution you'd like.

—

### Describe alternatives you've considered.

—

**Comment by eikemeier** [comment]
How is this different from [`inamedparam`](https://golangci-lint.run/usage/linters/#inamedparam)?
